лабораторная работа 2/lab_2.3.py

```diff
@@ -0,0 +1,65 @@
+# TODO  Напишите функцию count_letters
+
+
+def count_letters(text):
+    letter_count = {}
+
+    for char in text.lower():
+        if char.isalpha():
+            if char in letter_count:
+                letter_count[char] += 1
+            else:
+                letter_count[char] = 1
+
+        return letter_count
```

Comment:
Хорошее решение!
Но нужно немного подправить Ваш код.
Сейчас код не выводит нужного результата.
Это связано с лишней табуляцией в 14 строке.
В функции `count_letters `возврат значения (`return`) находится внутри цикла `for`, а значит, функция вернёт значение после первой итерации и завершится. Чтобы исправить эту ошибку, нужно переместить строку с `return` за пределы цикла `for`.
```suggestion
    return letter_count
```